Make the infobox expandable #102

Abbe98 opened the pull request:
# Description

Display official blog URL and official jobs URL (#287) _as a start_.

## Issue Reference

<!-- Link to the related issue in the repository. -->
Fixes #102 #287

## PR Details

### Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Documentation update
- [ ] Code refactor

## Checklist

Please **ensure** the following before submitting the PR:

- [x] I have **tested** the changes locally, and they work as expected.
- [x] The code follows the project's coding standards and conventions.

## Screenshots (applicable to user interface changes)

<!-- Attach screenshots if relevant. -->

[Screencast from 2024-12-08 17-06-18.webm](https://github.com/user-attachments/assets/9754d99a-fde9-4739-93a6-adb41bc5566d)
[Screencast from 2024-12-08 17-05-58.webm](https://github.com/user-attachments/assets/25b61027-eef4-4b3a-9aa5-366495d7c659)

RVA2869 commented on templates/org.html:
```diff
@@ -92,6 +92,20 @@ <h1 lang="{{ .orgLabel.Lang }}">{{ .orgLabel }}</h1>
+                    <hr>
+                    <ul>
+                        <li>Official jobs page: <span class="text-grey" id="additional-link-jobs">[missing data]</span></li>
+                        <li>Official blog: <span class="text-grey" id="additional-link-blog">[missing data]</span></li>
```
Suggestion: omit the `[missing data]` from these 2. Because it is not very common those links and if there is something “missing” **even if there is none** does not come across very well in my opinion.

Ainali replied:
I don't agree. This is an invitation to help add the data. And these are just the first few things we add, just with all open issues there may be 3-4 more.

Abbe98 replied:
I'm not a huge fan of this pattern either as a user can see the expand button just to get disappointed. However, people looking to contribute might not even know what they can contribute if we do not indicate this, this is for example a known problem with the call to action buttons. For example if you go to an agency missing an email address there is no way to know from that page that adding an email address will add a button.

For the future I imagine these "missing data" sections(both here and in the existing dialog) to be either accompanied with or replaced by "add"/"+" buttons which would open a modal and let people authenticate with OAuth and paste a new link.

Ainali commented:
I was trying to build locally but got this error:

```
Building project with 47 views.
Received bad(HTTP: 502 Bad Gateway) response from SPARQL endpoint:
<html><body><h1>502 Bad Gateway</h1>
The server returned an invalid or incomplete response.
</body></html>

Error: Failed to render page at site/statistics/index.html Error: template: statistics.html:37:35: executing "main" at <query "organization-optional.rq" .qid.String>: error calling query: Received bad response from SPARQL endpoint
```

Abbe98 commented:
> I was trying to build locally but got this error:
> 
> ```
> Building project with 47 views.
> Received bad(HTTP: 502 Bad Gateway) response from SPARQL endpoint:
> <html><body><h1>502 Bad Gateway</h1>
> The server returned an invalid or incomplete response.
> </body></html>
> 
> Error: Failed to render page at site/statistics/index.html Error: template: statistics.html:37:35: executing "main" at <query "organization-optional.rq" .qid.String>: error calling query: Received bad response from SPARQL endpoint
> ```

Looks like WDQS is having a bad day.

Ainali reviewed (approved):
Worked on the second try!

(Not a blocker, but if in the future the expanded view gets long, we might want to have an option to collapse it again. But let's merge this for now.)

Abbe98 commented:
Converted to draft until I have changed it so that the "expand" button only renders when there is additional data to be displayed. I will also create an issue about a generic "[missing data]"  (or something positive? "add information"?) replacement.

Abbe98 commented:
I'm scratching my head a little bit regarding the design here. If we only want to display this if there is information we end up in a situation in which we either show a loading indicator and a "fail message" or we end up with a jumpy behavior when the "expand" renders. I'm not a fan of either.

Maybe if we first turn the [missing data] messages into a call to action "[add information->]" it's more acceptable to always show this even if there is no additional information?
